chapi/chapiclient.go

```diff
@@ -271,7 +271,7 @@ func (chapiClient *Client) AttachDevice(volumes []*model.Volume) (devices []*mod
 		log.Errorf("AttachDevice: Err:%s for volume(%s)", err.Error(), volumes[0].Name)
 		return nil, err
 	}
-	
+	// Win CHAPI doesnt return Device[0].State, added OR condition to handeled it.	
```

Comment:
So the only change is a comment?

 - fix doesn't
 - fix handled

Reply:
Done. thanks.